DirectXTex/DirectXTexConvert.cpp

```diff
@@ -1659,9 +1659,7 @@ bool DirectX::Internal::StoreScanline(
             for (size_t icount = 0; icount < (size - sizeof(XMHALF4) + 1); icount += sizeof(XMHALF4))
             {
                 if (sPtr >= ePtr) break;
-                XMVECTOR v = *sPtr++;
-                v = XMVectorClamp(v, g_HalfMin, g_HalfMax);
-                XMStoreHalf4(dPtr++, v);
+                XMStoreHalf4(dPtr++, *sPtr++);
```

Comment:
I'm a little nervous about removing the clamp behavior here, but it's probably the best choice.

The main places where the clamping is now 'missing' is:

* BC Decompression - this shouldn't ever have specials
* TransformImage - The pixel function used needs to clamp if they care
* ConvertToRGBA32 helper - not using float16 formats as the target
* Custom resize, Custom mip generation, CopyRectange, NormalMap generation, PM Alpha conversion - If the input has specials, the output will have them.

I think the ``ConvertFromR32G32B32A32`` helpers needs an explicit clamp added for the float16 cases. This is the primarily used for WIC interactions where WIC doesn't support the format.

Reply:
>     * BC Decompression - this shouldn't ever have specials

BC6H signed can have -INF, even though it's not recommended.